lib/matplotlib/tests/test_widgets.py

```diff
@@ -619,7 +619,11 @@ def test_CheckButtons():
     check.disconnect(cid)
 
 
-def test_TextBox():
+@pytest.mark.parametrize("toolbar", ["none", "toolbar2", "toolmanager"])
+def test_TextBox(toolbar):
+    # Avoid "toolmanager is provisional" warning.
+    dict.__setitem__(plt.rcParams, "toolbar", toolbar)
```

Comment:
Out of interest, why this and not `plt.rcParams['toolbar'] = toolbar`?

Reply:
Because `rcParams["toolbar"] = "toolmanager"` emits a "toolmanager is provisional API" warning.  (Sure, you can play with warnings filters to suppress that, but ugh...)